kevlar/__init__.py

```diff
@@ -155,24 +148,4 @@ def vcf_header(outstream, version='4.2', source='kevlar', infoheader=False):
           sep='\t', file=outstream)
 
 
-def bwa_align(cmd, seqstring=None):
```

Comment:
This was migrated to the `reference` module recently, but somehow didn't get deleted from the package config.